packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -39,11 +39,23 @@ import { Omit } from '@uifabric/utilities';
 import { Point } from '@uifabric/utilities';
 import * as PropTypes from 'prop-types';
 import * as React from 'react';
+import { Rectangle } from '@uifabric/utilities';
 import { Selection } from '@uifabric/utilities';
 import { SELECTION_CHANGE } from '@uifabric/utilities';
 import { SelectionDirection } from '@uifabric/utilities';
 import { SelectionMode } from '@uifabric/utilities';
 
+// @public (undocumented)
+export const __positioningTestPackage: {

```

Comment:
What's this?

Reply:
This is why I really don't want to export these from the index.

Reply:
I also do not want to export individual things here as I don't know what is or isn't used by the partner.

Reply:
I'm gonna keep going here though. I've updated this to not export the test exports.

Reply:
Fixed.